.prettierrc.cjs

```diff
@@ -0,0 +1,7 @@
+/** @type {import('prettier').Config} */
```

Comment:
Let's use whatever settings don't lead to a massive diff when ran, lol. This looks good to me.

Reply:
too much to type. Will discuss why this file is necessary in the call